packages/core/src/utils.js

```diff
@@ -1230,10 +1230,11 @@ export function getMatchingOption(formData, options, rootSchema) {
       // been filled in yet, which will mean that the schema is not valid
       delete augmentedSchema.required;
 
-      if (isValid(augmentedSchema, formData, rootSchema)) {
+      if (formData && isValid(augmentedSchema, formData, rootSchema)) {
```

Comment:
Thanks. Can you add a test that covers this case, so it's clearer what issue this is fixing?